clang/include/clang/ExtractAPI/API.h

```diff
@@ -444,6 +567,24 @@ struct SymbolReference {
   bool empty() const { return Name.empty() && USR.empty() && Source.empty(); }
 };
 
+struct StaticFieldRecord : APIRecord {
```

Comment:
This shares a lot of similarities with GlobalVariables

Reply:
Just changed this to inherit from CXXFieldRecord, static fields also have access.